dotcom-rendering/src/model/front-schema.json

```diff
@@ -764,6 +764,32 @@
                                                             "properties": {
                                                                 "imageSrc": {
                                                                     "type": "string"
+                                                                },
+                                                                "assets": {
+                                                                    "type": "array",
+                                                                    "items": {
+                                                                        "type": "object",
+                                                                        "properties": {
+                                                                            "imageSrc": {
+                                                                                "type": "string"
+                                                                            },
+                                                                            "imageSrcWidth": {
+                                                                                "type": "string"
+                                                                            },
+                                                                            "imageSrcHeight": {
+                                                                                "type": "string"
+                                                                            },
```

Comment:
I noted these properties have come through on the schema despite not being on the model in `front.ts` - Does this need to be re-generated?